File: Function/6.py
input_id = input("Request ID.\n")
def login(_id):
    members = ['egoing','k8805','leezche']
    for member in members:
        if _id == member:
            return True
    return False
if login(input_id):
    print('Hello, '+input_id)
else:
    print('who are you?')

File: Function/6.rb
puts("Request ID")
input_id = gets.chomp()

def login(_id)
  members = ['egoing', 'k8805', 'leezche']
  for member in members do
      if member == _id
        return true
      end
  end
  return false
end

if login(input_id)
      puts('Hello, '+input_id)
else
  puts('who are you?')
end
